src/main/java/se/sundsvall/digitalmail/api/DigitalMailResource.java

```diff
@@ -30,89 +33,96 @@
+			@ApiResponse(responseCode = "400", description = "Bad Request", content = @Content(schema = @Schema(oneOf = {Problem.class, ConstraintViolationProblem.class}))),
+			@ApiResponse(responseCode = "404", description = "Not Found", content = @Content(schema = @Schema(implementation = Problem.class))),
+			@ApiResponse(responseCode = "500", description = "Internal Server Error", content = @Content(schema = @Schema(implementation = Problem.class)))
+		}
+	)
+	@PostMapping(
+		value = "/send-digital-mail",
+		consumes = APPLICATION_JSON_VALUE,
+		produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE}
+	)
+	ResponseEntity<DigitalMailResponse> sendDigitalMail(
+		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
+		@Valid @RequestBody final DigitalMailRequest request) {
+
+		// Validate body as HTML if content type is text/html
+		Optional.ofNullable(request.getBodyInformation())
+			.filter(bodyInfo -> TEXT_HTML_VALUE.equals(bodyInfo.getContentType()) && !htmlValidator.validate(bodyInfo.getBody()))
+			.ifPresent(bodyInfo -> {
+				throw Problem.builder()
+					.withTitle("Body HTML is invalid")
+					.withStatus(BAD_REQUEST)
+					.withDetail("Use https://validator.w3.org/ to make sure your HTML validates")
+					.build();
+			});
+
+		final var response = digitalMailService.sendDigitalMail(new DigitalMailDto(request), municipalityId);
+		return ok(response);
+	}
+
+	@Operation(
+		summary = "Send a digital invoice",
+		responses = {
+			@ApiResponse(responseCode = "200", description = "Successful Operation", useReturnTypeSchema = true),
+			@ApiResponse(responseCode = "400", description = "Bad Request", content = @Content(schema = @Schema(oneOf = {Problem.class, ConstraintViolationProblem.class}))),
+			@ApiResponse(responseCode = "404", description = "Not Found", content = @Content(schema = @Schema(implementation = Problem.class))),
+			@ApiResponse(responseCode = "500", description = "Internal Server Error", content = @Content(schema = @Schema(implementation = Problem.class)))
+		}
+	)
+	@PostMapping(
+		value = "/send-digital-invoice",
+		consumes = APPLICATION_JSON_VALUE,
+		produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE}
+	)
+	ResponseEntity<DigitalInvoiceResponse> sendDigitalInvoice(
+		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
+		@Valid @RequestBody final DigitalInvoiceRequest request) {
+		return ok(digitalMailService.sendDigitalInvoice(new InvoiceDto(request), municipalityId));
+	}
+
+	@Operation(
+		summary = "Check if a party has a digital mailbox and if said party has chosen to receive digital mail",
+		responses = {
+			@ApiResponse(responseCode = "200", description = "Successful Operation (available mailbox(es) found)"),
```

Comment:
Not an issue, with your PR, but a question regarding the chosen status code for successful responses:
- Shouldn't this be "204 (no content)" instead?

Reply:
According to DIGG `204 No Content` should not be used for `POST` (or `GET`).

In my opinion this should be a `GET` with `200 OK`.